docs-src/cloud-context/saml-finish.md

```diff
@@ -0,0 +1,19 @@
+---
+id: saml-finish
+title: How to finish your SAML configuration
+sidebar_label: Finish SAML configuration
+description: To finish your SAML configuration, send the sign-in URL and certificate to us and then test your connection.
+tags:
+  - guide-context
+---
+
+After you configure SAML with your IdP, we can finish the configuration on our side.
+To do that, send the following information to us:
+
+- The sign-in URL from your application
+- The SAML sign-in certificate
```

Comment:
Please include the "IdP Domain" as a requirement--and note they can provide multiple